tests/direct_messaging_test.go

```diff
@@ -28,6 +28,10 @@ func TestWriteMDirectAccountData(t *testing.T) {
 
 	alice := deployment.Client(t, "hs1", "@alice:hs1")
 	bob := deployment.Client(t, "hs1", "@bob:hs1")
+
+	// additional logging to debug https://github.com/matrix-org/synapse/issues/13334
+	alice.Debug = true
+
```

Comment:
Was gonna ask if we wanna do the same thing for `bob`, but we never make any client-server requests on his behalf.

Reply:
yeah, the problems all seem to be with requests from alice.